Modify `BlobSidecar` for Gloas (#4846)
ethereum/consensus-specs: specs/gloas/p2p-interface.md

### Containers

#### Modified `BlobSidecar`

*Note*: The `signed_block_header` and `kzg_commitments_inclusion_proof` fields

> have been removed from `BlobSidecar` in Gloas as header and inclusion proof

Reviewer: ```suggestion
have been removed from `DataColumnSidecar` in Gloas as header and inclusion proof
```

proof verifications are no longer required in ePBS. Instead, sidecars are

> validated by checking that the hash of `kzg_commitments` matches what's

Reviewer 2: Just a nit. I prefer not to use contractions.
```suggestion
validated by checking that the hash of `kzg_commitments` matches what is
```

committed in the builder's bid for the corresponding `beacon_block_root`.

> ``` python

Reviewer 2: ```suggestion
```python
```
